analyzer/python/ikos/scan.py

```diff
@@ -584,12 +585,23 @@ def extract_bitcode(exe_path, bc_path):
 
 def notify_binary_built(exe_path, bc_path):
     ''' Notify the scan server that a binary was built '''
-    binary = {
-        'exe': os.path.abspath(exe_path),
-        'bc': os.path.abspath(bc_path),
-    }
-    data = http.urlencode(binary).encode('utf-8')
-    http.urlopen(os.environ['IKOS_SCAN_SERVER'], data)
+    abs_bc_path = os.path.abspath(bc_path)
+    abs_exe_path = os.path.abspath(exe_path)
+
+    if 'IKOS_SCAN_NOTIFIER_FILES' in os.environ:
+        bc_base_path, _ = os.path.splitext(abs_bc_path)
+        indicator_path = bc_base_path + '.ikosbin'
+        with open(indicator_path, 'w') as indicator_file:
+            indicator_file.write(json.dumps({
+                'exe': abs_exe_path,
+                'bc': abs_bc_path}))
```

Comment:
nitpick: could you keep the formatting consistent?
```suggestion
            indicator_file.write(json.dumps({
                'exe': abs_exe_path,
                'bc': abs_bc_path,
            }))
```